translations/deckbuilding.tex/pl.po

```diff
@@ -243,13 +330,24 @@ msgid ""
 "Spell.  All Spell Cards also have an alternative bottom (5) "
 "\\includesvg[height=10px]{\\svgs/empower.svg} effect."
 msgstr ""
+"Karty Zaklęć\\index{Karta Zaklęcia} mają trzy możliwe efekty podstawowe."
+"Użycie górnej, podstawowej wersji Zaklęcia nie ma dodatkowych kosztów."
```

Comment:
Dodałbym, że użycie podstawowej wersji zaklęcia „zwykle” nie ma dodatkowych kosztów. Wydaje mi się, że są zaklęcia, takie jak Implozja, które od początku wymagają wzmocnienia. Ale mogę się mylić.

Reply:
Tu też proponowałbym issue. Mimo, że chętnie bym to zmienił, to jeśli mielibyśmy bardziej precyzować, zaczynajmy od wersji en